packages/core/src/Stores/ui-store.js

```diff
@@ -110,6 +110,9 @@ export default class UIStore extends BaseStore {
     // UI Focus retention
     @observable current_focus = null;
 
+    // Enabling EU users
+    @observable is_eu_enabled = true;
```

Comment:
This seems to not be modified anywhere, is it a constant or is this preliminary work for actions later (before merge of `eu` branch to `dev`)?

Reply:
@mahdipourziaei-binary it is constant. actually, it is a temporary config for enabling and disabling EU parts for limit. when all EU parts are done, we can remove it

Reply:
@mahdiaryayi-fs I've added a TODO on it then, thanks.

Reply:
@mahdipourziaei-binary TODO is added. please approve if nothing left.